### DISCUSSION TITLE
安装过程中出现 CGROUPS_CPU: missing  什么问题呀

### DISCUSSION THREAD

**Discussion by lkzwc** [kubesphere/kubesphere#5773] [Q&A]
![image](https://github.com/kubesphere/kubesphere/assets/84896877/4d81c2e1-5d72-4be0-82af-457ea7e06559)

**Comment by lkzwc** [comment]
OS :Rocky Linux 9.0 64位.之前是部署成功过的

**Comment by lkzwc** [comment]
![image](https://github.com/kubesphere/kubesphere/assets/84896877/cf9dfee0-77ab-49d4-86e0-15dfcfe3cd31)
没有人 看看这个问题吗

**Reply by binary01010** [reply]
换个常用linux发行版本，比如centos Ubuntu 等，或者检查一下内核版本是否支持cgroup v2

**Comment by Aurorxa** [comment]
① echo "cgroup_memory=1 cgroup_enable=memory" >> /boot/cmdline.txt 
② reboot
